dags/hivemind_etl_helpers/src/db/github/extract/github_extraction.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+import neo4j
+from hivemind_etl_helpers.src.db.github.extract.issues import GithubIssueExtraction
+from hivemind_etl_helpers.src.db.github.schema import GitHubIssue
+
+
+class GithubExtraction:
+    def __init__(self):
+        # to be uncommented once other pull requests
+        # regarding `extraction` are ready
+        # self.commits_extraction = GithubCommitExtraction()
+        # self.pull_requests_extraction = GithubPullRequestsExtraction()
+        # self.comment_extraction = GitHubCommentExtraction()
+        self.issue_extraction = GithubIssueExtraction()
+
+    def _fetch_raw_issues(
```

Comment:
Was there any specific reason we included the `_fetch_raw_issues` within GitHubExtraction?

I think having the fetch_issues and fetch_issue_ids would be enough in our `GitHubExtraction`

Reply:
Currently, the _raw methods are still called from GitHubExtraction inside the test cases.

Reply:
Ah, yes I see your point. To access that in test cases you can call the `_fetch_raw_issues` directly from `GitHubExtraction`, you can do the following within test cases.

```python
self.extractor.issue_extraction._fetch_raw_issues
```